File: 01. DataStructures/02. Stack/LeetCode Questions/06. Crawler Log Folder #1598/Solution.java
class Solution {
    public int minOperations(String[] logs) {
        Stack st = new Stack<String>();
        for(String i : logs){
            if(i.equals("../")){
                if(!st.empty())
                    st.pop();
            }
            else if(i.equals("./")) 
                continue;
            else
                st.push(i);
        }
        return st.size();
    }
}

File: 01. DataStructures/02. Stack/LeetCode Questions/06. Crawler Log Folder #1598/solution.js
var minOperations = function(logs) {

    let output = 0;
    logs.forEach((log) => {
        if(log == "../") {
            output -= 1;
        } else if(log !== "./"){
            output +=  1;
        }
        
        if(output < 0) {
            output = 0;
        }
    });
    
    
    return output;
};

console.log(minOperations(["./","wz4/","../", "../", "../", "../", "../"]));
